source/endpoints/jobs.rst

```diff
@@ -0,0 +1,153 @@
+*************
+Jobs
+*************
+
+``GET /jobs/{id}``
+*******************
+
+Gets an existing transcription job by id
+
+**CURL Examples**
+
+.. code:: javascript
+
+  curl -X GET "https://api.rev.ai/revspeech/v1beta/jobs/{id}" -H "Authorization: Bearer <api_key>"
+
+**Parameters**
+
+====================== ===============================================================
+Name                   Description
+====================== ===============================================================
+id ``(required)``      Gets an existing transcription job by id
+====================== ===============================================================
+
+**Responses**
+
+====================== ===============================================================
+Code                   Description
+====================== ===============================================================
+200                    Revspeech API Job
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                          "id": "111111",
+                          "status": "in_progress",
+                          "created_on": "2018-05-05T23:23:22.29Z"
+                        }          
+---------------------- ---------------------------------------------------------------
+401                    Request Unauthorized
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                          "title": "Authorization has been denied for this request"
+                        }    
+---------------------- ---------------------------------------------------------------
+404                    Job Not Found
+====================== ===============================================================
+
+
+``POST /jobs``
+*****************
+
+Submits a transcription job
+
+**CURL Examples**
+
+.. code:: javascript
+
+    curl -X POST "https://api.rev.ai/revspeech/v1beta/jobs" -H "Authorization: Bearer <api_key>" -H "Content-Type: application/json" -d "{\"media_url\":\"https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3\",\"metadata\":\"This is a sample submit jobs option\"}"
+
+.. code:: javascript
+
+  curl -X POST "https://api.rev.ai/revspeech/v1beta/jobs" -H "Authorization: Bearer <api_key>" -H "Content-Type: multipart/form-data" -F "media=@media_file.mp3" -F "options={"metadata":"This is a sample submit jobs option for multipart"}"
+
+**Request Body (required)**
+
```

Comment:
I think it would be useful to have some description here explaining that this represents two options - uploading media inline via multipart and specifying link to the media (in the application/json option). its otherwise not exactly clear what is going on.